lesson-7/todos/src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom'
+
+import { createStore, applyMiddleware } from 'redux'
+import { Provider } from "react-redux";
+import logger from 'redux-logger'
+
+import './styles/index.css';
+import App from './components/App';
+import registerServiceWorker from './registerServiceWorker';
+
+import Reducer from './Reducer';
+
+import thunk from 'redux-thunk';
+
+let store = createStore(Reducer, 
+    applyMiddleware(logger),
+    applyMiddleware(thunk)
```

Comment:
We haven't touched thunk nor saga at all. Not sure if we want to add it now. I think the best would be to do it without it or at least just quickly mention it. But not to dive into it I guess.

Reply:
I agree. I believe it could be too much adding sagas to this lesson 😢